commit 91bdb202c2d840ff0e4f491859b10fd5ae8721fd
Author: Alexandre Belloni <alexandre.belloni@free-electrons.com>
Date:   Thu Jan 19 23:05:39 2017 +0100

    ARM: dts: at91: Enable DMA on sama5d2_xplained console
    
    commit 78162d48466d23c45a784034630c5928af631e3d upstream.
    
    Enable DMA on uart1 to get a more reliable console.
    
    Acked-by: Nicolas Ferre <nicolas.ferre@atmel.com>
    Signed-off-by: Alexandre Belloni <alexandre.belloni@free-electrons.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1231_91bdb202c2d840ff0e4f491859b10fd5ae8721fd.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1231_91bdb202c2d840ff0e4f491859b10fd5ae8721fd.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
